client.go

```diff
@@ -177,6 +180,8 @@ func NewNativeClient(user, host, clientVersion string, port int, auth *Auth, hos
 		Config:        config,
 		Hostname:      host,
 		Port:          port,
+		ProxyHost:     proxyHost,
+		ProxyPort:     proxyPort,
```

Comment:
Would it make sense to also have a ProxyConfig here in case the proxy config and the final hop config need to be different?